inception-workload-dynamic/src/main/java/de/tudarmstadt/ukp/inception/workload/dynamic/support/AnnotationQueueOverviewDataProvider.java

```diff
@@ -162,39 +172,38 @@ public void detach()
         List<SourceDocument> dateList = new ArrayList<>();
         List<SourceDocument> unusedList = new ArrayList<>();
 
-        //Avoid error in one specific case
+        // Avoid error in one specific case
         if (filter.getSelected() == null) {
-            filter.setSelected("false");
+            filter.setSelected(Boolean.FALSE.toString());
```

Comment:
Why does this have to be a string? Why is it not a boolean field in the Filter class?